api/main_endpoints/routes/3DPrintingForm.js

```diff
@@ -3,8 +3,7 @@
 const express = require('express');
 const router = express.Router();
 const PrintingForm3D = require('../models/PrintingForm3D.js');
-const settings = require('../util/settings');
-const logger = require(`${settings.util}/logger`);
```

Comment:
just wondering, why was this removed?

Reply:
`logger` is a file from a long time ago which would write limited logs to a file in Core-v4. It isn't used very much and that's why i took it out